packages/wrangler/src/sites.ts

```diff
@@ -116,84 +128,108 @@ export async function syncAssets(
 	if (siteAssets === undefined) {
 		return { manifest: undefined, namespace: undefined };
 	}
-
 	if (dryRun) {
 		logger.log("(Note: doing a dry run, not uploading or deleting anything.)");
 		return { manifest: undefined, namespace: undefined };
 	}
 	assert(accountId, "Missing accountId");
 
+	// Create assets namespace if it doesn't exist
 	const title = `__${scriptName}-workers_sites_assets${
 		preview ? "_preview" : ""
 	}`;
 
+	logger.info("Fetching list of already uploaded assets...");
```

Comment:
Is this log in the correct place? Shouldn't it go after the call to `createKVNamespaceIfNotAlreadyExisting()`?